regression/goto-instrument/nondet-volatile-02/test.desc

```diff
@@ -0,0 +1,16 @@
+CORE
+test.c
+--nondet-volatile-variable b --nondet-volatile-variable func::c --nondet-volatile-variable main::1::d
+\[func.assertion.1\] line \d+ assertion c == 0: FAILURE
+\[func.assertion.2\] line \d+ assertion c != 0: FAILURE
+\[main.assertion.1\] line \d+ assertion a == 0: SUCCESS
+\[main.assertion.2\] line \d+ assertion b == 0: FAILURE
+\[main.assertion.3\] line \d+ assertion b != 0: FAILURE
+\[main.assertion.4\] line \d+ assertion d == 0: FAILURE
+\[main.assertion.5\] line \d+ assertion d != 0: FAILURE
+^EXIT=10$
+^SIGNAL=0$
+--
+--
+Check that volatile global, local, and parameter variables are havocked when
+given via the --nondet-volatile-variable option
```

Comment:
:pick add to description - `a is not havoc-d so the assert succeeds`

Reply:
Added